packages/adapter-netlify/README.md

```diff
@@ -36,6 +36,27 @@ Then, make sure you have a [netlify.toml](https://docs.netlify.com/configure-bui
 
 It's recommended that you add the `build` and `functions` folders (or whichever other folders you specify) to your `.gitignore`.
 
-## Extra Resources
-
-Guide for SvelteKit + Netlify Forms users: https://dev.to/swyx/how-to-use-sveltekit-with-netlify-forms-5gmj
+## Troubleshooting
+
+### How to make Netlify forms work?
+
+1. Add the following [prerendering configuration](https://kit.svelte.dev/docs#configuration-prerender) to `svelte.config.js`:
+   ```js
+   prerender: {
+       crawl: true,
+       enabled: true,
+       force: true,
+       pages: ['*']
+   }
+   ```
+2. Create your Netlify HTML form as described [here](https://docs.netlify.com/forms/setup/#html-forms), e.g. as `/routes/contact.svelte`. (Don't forget to add the hidden "form-name" input element!)
+3. Tell SvelteKit to prerender your `contact.svelte` page component by adding:
+   ```svelte
+   <script context="module">
+       export const prerender = true;
+   </script>
+   ```
+4. If your Netlify form has a [custom success message](https://docs.netlify.com/forms/setup/#success-messages), e.g. something like `<form name="contact" netlify method="POST" action="/success">`, then:
+   - Ensure `/success` points to a proper Svelte page component
+   - In it add the same prerender hint as in step 3
+5. That's it! _(You can find some more details in the related [GitHub discussion](https://github.com/sveltejs/kit/issues/942#issuecomment-828726791).)_
```

Comment:
let's not link to the issue. everything you need to know should be explained here. if there's something that people might want to know that's currently in the issue then we can move it here